server/src/main/java/io/spine/server/bus/Listeners.java

```diff
@@ -51,4 +52,9 @@ public void accept(E envelope) {
             }
         });
     }
+
+    @VisibleForTesting
+    boolean contains(Consumer<E> listener) {
+        return listeners.contains(listener);
```

Comment:
Can we create a distinct interface for a listener? Maybe not the scope of this PR though.

A `Consumer` seems rather raw and generic. `interface Listener<E> extends Consumer<E>` would at least give us a place to explain the purpose of a listener.